src/scraper.py

```diff
@@ -0,0 +1,47 @@
+import feedparser
+
+AUTHORIZED_TOPICS = [
+    "WORLD",
+    "NATION",
+    "BUSINESS",
+    "TECHNOLOGY",
+    "ENTERTAINMENT",
+    "SPORTS",
+    "SCIENCE",
+]
+
+class GoogleNewsScraper:
+    def __init__(self):
+        self.lang_suffix = "?hl=fr&gl=FR&ceid=FR:fr"
+        self.query_suffix_template = "search?q={query}"
+        self.base_url = "https://news.google.com/rss"
+        self.topic_url_template = self.base_url + "/" + "headlines/section/topic/{topic}"
+
+
+    def top_news(self) -> list[str]:
+        url = self.base_url + self.lang_suffix
+        retrieved_urls = self.retrieve_urls(url)
+        return retrieved_urls
+        
+
+    def query(self, query: str) -> list[str]:
```

Comment:
dead code ? i don't see the usage

Reply:
Not used yet, but we could design a more elaborated `scripts/scraper.py` that uses this method. Should I remove it ?

Reply:
following YAGNI principle, i woule say that we can add it back if and when we need it
https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it